vent/menus/main.py

```diff
@@ -350,6 +354,14 @@ def create(self):
                                  labelColor='DEFAULT', value=Cpu())
         self.gpufield = self.add(npyscreen.TitleFixedText, name='GPUs:',
                                  labelColor='DEFAULT', value=Gpu()[1])
+        self.location = self.add(npyscreen.TitleFixedText,
+                                 name='.vent:',
+                                 value=PathDirs().meta_dir,
+                                 labelColor='DEFAULT')
+        self.file_drop = self.add(npyscreen.TitleFixedText,
+                                  name='File Drop:',
+                                  value=template.section("main")[1][0][1],
```

Comment:
what i meant was you can't guarantee that `files` is the only option (or the first one) in the `main` section.  eventually there will be other options.  just having the path (value) is good, but it needs to specifically grab that option/value.